client/layout/guided-tours/docs/API.md

```diff
@@ -216,3 +216,53 @@ combineTours( {
   thirdTour: ThirdTour,
 } );
 ```
+
+## Targeting elements in Calypso
+
+In few places in GT you need to target a DOM elements in Calypso. There are currently two ways to do this:
+
+- `data-tip-target` attribute on the target element
+- using any css selector
+
+To find the DOM node, `document.querySelector` is used. By default, the query used is `[data-tip-target="${target}"]`. However, if `target` starts with a dot or a space character, it will be treated like a standard selector and be passed directly to the function, thereby allowing you to target elements that do no set a `data-tip-target` attribute.
+
+### [data-tip-target]
+
+You mark a target by adding `data-tip-target="some-name"` attribute to it with your value. Please be specific to avoid conflicts and use dash-case.
+
+In your tour, just put a value of your desired `[data-tip-target]` into the `target` prop.
+
+Example: you want to position a step of your tour to point to some input element.
+
+```jsx
+// code somewhere in Calypso
+<input type="text" name="example" value={ … } data-tip-target="my-example-input" />
+
+// in your tour
+<Step name="example-step" target="my-example-input">…</Step>
+```
+
+### CSS selector
+
+There are cases when it might be a better idea to target an element by a CSS selector.
```

Comment:
See my previous comment on this.

> There are cases when

Which? :)

_Edit:_ I've now read the examples at the bottom, and a couple of them do reveal the advantages of the selector method. Should we make that explicit, and more towards the top of the section?